config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -135,11 +134,11 @@ presubmits:
           privileged: true
         resources:
           limits:
-            cpu: 4
-            memory: 9Gi
+            cpu: 7
```

Comment:
this makes a huge difference
https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/126563/pull-kubernetes-e2e-kind-canary/1926043087319076864
has been testing for some time now

https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/126563/pull-kubernetes-e2e-kind-ipv6-canary/1926043087381991424
is still compiling ... 18 minutes in